src/playlist-selectors.js

```diff
@@ -190,6 +193,16 @@ export const simpleSelector = function(master,
     (rep) => rep.bandwidth === highestRemainingBandwidthRep.bandwidth
   )[0];
 
+  // if we are ignoring the player size, make an early decision
+  if (ignorePlayerSize) {
+    let chosenRep = (
+      bandwidthBestRep ||
+      enabledPlaylistReps[0] ||
+      sortedPlaylistReps[0]
+    );
```

Comment:
You'll need a newline here to fix linting errors

Reply:
yea so like I got a *ton* of linting errors relating to `exceeds the maximum line length of 90`, `Don't make functions within a loop` and `Unexpected "todo" comment` so I admittedly ignored all of them and just opened a PR... sorry

Reply:
No worries, when we update our linter I think it should be more clear which of those is a linter error and just a warning